superqt/fonticon/_plugins.py

```diff
@@ -0,0 +1,83 @@
+from typing import Set, Tuple
+
+from ._iconfont import IconFontMeta, namespace2font
+
+try:
+    from importlib.metadata import EntryPoint, entry_points
+except ImportError:
+    from importlib_metadata import EntryPoint, entry_points  # type: ignore
+
+
+class FontIconManager:
+
+    ENTRY_POINT = "superqt.fonticon"
+    _PLUGINS: dict[str, EntryPoint] = {}
+    _LOADED: dict[str, IconFontMeta] = {}
```

Comment:
ah, right... it should be `Dict`.  It's a long story... the lowercase form was added by the `pyupdgrade` pre-commit step before I added `--keep-runtime-typing`.  it can work in many cases... but I generally try not to use that form yet and this one slipped through.  thanks!